tests/validate/test_validate_merge_logic.py

```diff
@@ -7,42 +7,42 @@ def test_validate_merge_logic(spark_session):
     """Test that validator works to assign failed flags to columns where result is not unique"""
     df = spark_session.createDataFrame(
         data=[
-            ("ONS00000001", 1, None, None, None, None, None),
-            ("ONS00000002", 1, None, None, None, None, None),
-            ("ONS00000002", 1, 1, 1, None, None, None),
-            ("ONS00000003", None, None, None, 1, None, 1),
-            ("ONS00000003", None, None, None, 1, None, 1),
-            ("ONS00000004", None, None, None, 1, 1, 1),
-            ("ONS00000004", None, None, None, 1, 1, 1),  # check failed flags cannot apply to dropped columns
-            ("ONS00000005", 1, None, 1, None, None, None),
-            ("ONS00000005", 1, None, 1, None, None, None),
-            ("ONS00000005", 1, None, 1, None, None, None),  # check correct failed flags detected
-            ("ONS00000006", None, None, None, None, None, None),
-            ("ONS00000006", None, None, None, None, None, None),  # check a match has been applied
-            ("ONS00000007", 1, None, 1, 1, None, 1),  # check cannot flag as more than 1 type
+            ("ONS00000001", 1, None, None, None, None),
```

Comment:
There's another output column called failed1 that needs to be deleted from input